src/renderers/dom/shared/hooks/possibleStandardNames.js

```diff
@@ -0,0 +1,309 @@
+  'strikethrough-position': 'strikethroughPosition',
+  strikethroughposition: 'strikethroughPosition',
+  strikethroughthickness: 'strikethroughThickness',
+  'strikethrough-thickness': 'strikethroughThickness',
+  'stroke-dasharray': 'strokeDasharray',
+  strokedasharray: 'strokeDasharray',
+  'stroke-dashoffset': 'strokeDashoffset',
+  strokedashoffset: 'strokeDashoffset',
+  'stroke-linecap': 'strokeLinecap',
+  strokelinecap: 'strokeLinecap',
+  'stroke-linejoin': 'strokeLinejoin',
+  strokelinejoin: 'strokeLinejoin',
+  'stroke-miterlimit': 'strokeMiterlimit',
+  strokemiterlimit: 'strokeMiterlimit',
+  'stroke-opacity': 'strokeOpacity',
+  strokeopacity: 'strokeOpacity',
+  'stroke-width': 'strokeWidth',
+  surfacescale: 'surfaceScale',
+  systemlanguage: 'systemLanguage',
+  tablevalues: 'tableValues',
+  targetx: 'targetX',
+  targety: 'targetY',
+  'text-anchor': 'textAnchor',
+  textanchor: 'textAnchor',
+  'text-decoration': 'textDecoration',
+  textdecoration: 'textDecoration',
+  'text-rendering': 'textRendering',
+  textrendering: 'textRendering',
+  textlength: 'textLength',
+  'underline-position': 'underlinePosition',
+  underlineposition: 'underlinePosition',
+  'underline-thickness': 'underlineThickness',
+  underlinethickness: 'underlineThickness',
+  'unicode-bidi': 'unicodeBidi',
+  unicodebidi: 'unicodeBidi',
+  'unicode-range': 'unicodeRange',
+  unicoderange: 'unicodeRange',
+  'units-per-em': 'unitsPerEm',
+  unitsperem: 'unitsPerEm',
+  'v-alphabetic': 'vAlphabetic',
+  valphabetic: 'vAlphabetic',
+  'v-hanging': 'vHanging',
+  vhanging: 'vHanging',
+  'v-ideographic': 'vIdeographic',
+  videographic: 'vIdeographic',
+  'v-mathematical': 'vMathematical',
+  vmathematical: 'vMathematical',
+  'vector-effect': 'vectorEffect',
+  vectoreffect: 'vectorEffect',
+  'vert-adv-y': 'vertAdvY',
+  vertadvy: 'vertAdvY',
+  'vert-origin-x': 'vertOriginX',
+  vertoriginx: 'vertOriginX',
+  'vert-origin-y': 'vertOriginY',
+  vertoriginy: 'vertOriginY',
+  viewbox: 'viewBox',
+  viewtarget: 'viewTarget',
+  'word-spacing': 'wordSpacing',
+  wordspacing: 'wordSpacing',
+  'writing-mode': 'writingMode',
+  writingmode: 'writingMode',
+  'x-height': 'xHeight',
+  xheight: 'xHeight',
+  xchannelselector: 'xChannelSelector',
+  'xlink:actuate': 'xlinkActuate',
+  xlinkactuate: 'xlinkActuate',
+  'xlink:arcrole': 'xlinkArcrole',
+  xlinkarcrole: 'xlinkArcrole',
+  'xlink:href': 'xlinkHref',
+  xlinkhref: 'xlinkHref',
+  'xlink:role': 'xlinkRole',
+  xlinkrole: 'xlinkRole',
+  'xlink:show': 'xlinkShow',
+  xlinkshow: 'xlinkShow',
+  'xlink:title': 'xlinkTitle',
+  xlinktitle: 'xlinkTitle',
+  'xlink:type': 'xlinkType',
+  xlinktype: 'xlinkType',
+  'xml:base': 'xmlBase',
+  xmlbase: 'xmlBase',
+  'xmlns:xlink': 'xmlnsXlink',
+  xmlnsxlink: 'xmlnsXlink',
+  'xml:lang': 'xmlLang',
+  xmllang: 'xmlLang',
+  'xml:space': 'xmlSpace',
+  xmlspace: 'xmlSpace',
+  ychannelselector: 'yChannelSelector',
+  zoomandpan: 'zoomAndPan'
+}
```

Comment:
@jquense I think this should take care of attribute warning helpers. Though I'm going to work on ways to make this more concise :)